package.json

```diff
@@ -41,6 +41,6 @@
   "license": "",
   "name": "go-libp2p-raft",
   "releaseCmd": "git commit -a -m \"gx publish $VERSION\"",
-  "version": "0.0.2"
+  "version": "0.1.0"
```

Comment:
I mean, why is this changing and not lastPubVersion. And why a minor version bump if there is no new functionality?